<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qtGuiCVClass</class>
 <widget class="QMainWindow" name="qtGuiCVClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qtGuiCV</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" colspan="2">
     <widget class="QTextEdit" name="logTextEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>4</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>60</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="placeholderText">
       <string extracomment="log section..."/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="o_label">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>the original iamge</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="a_label">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>the converted image</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>885</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu12132">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="actionreopen_in_B_zone"/>
   </widget>
   <widget class="QMenu" name="menuOperation">
    <property name="title">
     <string>Operation</string>
    </property>
    <widget class="QMenu" name="menuMask_with_kernel">
     <property name="title">
      <string>CV Opthion</string>
     </property>
     <addaction name="actionshow_by_cv"/>
    </widget>
    <addaction name="action_toGray"/>
    <addaction name="actionMask_OP"/>
    <addaction name="menuMask_with_kernel"/>
    <addaction name="actionMask_with_kernel"/>
   </widget>
   <addaction name="menu12132"/>
   <addaction name="menuOperation"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action_New">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_OpenImg">
   <property name="text">
    <string>OpenImg</string>
   </property>
  </action>
  <action name="action_toGray">
   <property name="text">
    <string>&amp;toGray</string>
   </property>
  </action>
  <action name="actionMask_OP">
   <property name="text">
    <string>Mask OP</string>
   </property>
  </action>
  <action name="actionshow_by_cv">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>show by cv</string>
   </property>
  </action>
  <action name="actionMask_with_kernel">
   <property name="text">
    <string>Mask with kernel</string>
   </property>
  </action>
  <action name="actionreopen_in_B_zone">
   <property name="text">
    <string>reopen in B zone</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qtGuiCV.qrc"/>
 </resources>
 <connections/>
</ui>
